app/assets/javascripts/discourse/tests/acceptance/composer-test.js

```diff
@@ -956,6 +962,99 @@ acceptance("Composer", function (needs) {
       "it does not unescape script tags in code blocks"
     );
   });
+  function wait(timeout = 2000) {
```

Comment:
I don't see where this is used?

Reply:
Yeah. Thanks for spotting this! It shouldn't be here and was only added to debug the tests.